tests/virtctl/ssh.go

```diff
@@ -0,0 +1,84 @@
+package virtctl
+
+import (
+	"crypto/ecdsa"
+	"os"
+	"path/filepath"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	"golang.org/x/crypto/ssh"
+
+	"kubevirt.io/client-go/kubecli"
+	"kubevirt.io/kubevirt/tests"
+	"kubevirt.io/kubevirt/tests/clientcmd"
+	"kubevirt.io/kubevirt/tests/console"
+	"kubevirt.io/kubevirt/tests/libvmi"
+	"kubevirt.io/kubevirt/tests/util"
+)
+
+var _ = FDescribe("[sig-compute][virtctl]SSH", func() {
+	var pub ssh.PublicKey
+	var keyFile string
+	var virtClient kubecli.KubevirtClient
+	var kubevirtKnowHosts string
+
+	BeforeEach(func() {
+		var err error
+		virtClient, err = kubecli.GetKubevirtClient()
+		Expect(err).ToNot(HaveOccurred())
+		// Disable SSH_AGENT to not influence test results
+		Expect(os.Setenv("SSH_AUTH_SOCK", "/dev/null")).To(Succeed())
+		keyFile = filepath.Join(GinkgoT().TempDir(), "id_rsa")
+		kubevirtKnowHosts = filepath.Join(GinkgoT().TempDir(), "kubevirt_known_hosts")
+		tests.BeforeTestCleanup()
+		var priv *ecdsa.PrivateKey
+		priv, pub, err = NewKeyPair()
+		Expect(err).ToNot(HaveOccurred())
+		Expect(DumpPrivateKey(priv, keyFile)).To(Succeed())
+	})
+	AfterEach(func() {
+		err := os.RemoveAll(kubevirtKnowHosts)
```

Comment:
I think no need to remove that since the file is under `GinkgoT().TempDir()`.

Also you are not passing this to `--known-hosts=` arg in the tests. Is it intentional?

Reply:
oh this is a leftover of my previous experimenting. Thanks